### ISSUE TITLE
The operation couldn’t be completed. (CocoaPods.CPPodfileInitErrors error 0.)

### ISSUE THREAD

**Issue by dmalone08** [CocoaPods/CocoaPods#7676]
# Report

## What did you do?

ℹ I tried to create a New Podfile from Xcode Project. 

## What did you expect to happen?

ℹ I expected for the Install window to open.

## What happened instead?

ℹAn error window opened up saying "The operation couldn’t be completed. (CocoaPods.CPPodfileInitErrors error 0.)"

## CocoaPods Environment

ℹ CocoaPods : 1.4.0
        Ruby : ruby 2.2.6p396 (2016-11-15 revision 56800) [x86_64-darwin15]
    RubyGems : 2.6.8
        Host : Mac OS X 10.13.4 (17E199)
       Xcode : 9.3 (9E145)
         Git : git version 2.6.2

## Project that demonstrates the issue

ℹ https://github.com/dmalone08/my-app

**Comment by dmalone08** [comment]
Yes I'm using the app. I haven't been able to add a Podfile and I have been trying all day. I have uninstalled and reinstalled the app to no avail.

**Comment by dnkoutso** [comment]
@dmalone08 use command line?

**Comment by dmalone08** [comment]
Just the app. I haven't tried command line.

**Comment by dnkoutso** [comment]
I am uncertain. The app is in a different repo this is more about the library itself. Command line should work fine.

**Comment by dmalone08** [comment]
That did.
